src/deploymentmanager.ts

```diff
@@ -558,16 +561,16 @@ export class DeploymentManager implements IDeploymentManager {
             let cmd = '';
             switch ( os.type() ) {
                 case 'Windows_NT': {
+                    envvar = envvar.replace('=', ' ');
```

Comment:
can we write envvar into local file for Windows?

Reply:
Yes, we can have. I have opened a PBI for [it](https://msazure.visualstudio.com/b32aa71e-8ed2-41b2-9d77-5bc261222004/_workitems/edit/3143395).